MLS.Agent/CommandLine/VerifyCommand.cs

```diff
@@ -257,5 +271,28 @@ void SetOk()
                 }
             }
         }
+
+        private static bool ProjectIsCompatibleWithLanguage(string projectOrPackageName, string language)
+        {
+            var extenstion = Path.GetExtension(projectOrPackageName)?.ToLowerInvariant();
+            var supported = true;
+            if (!string.IsNullOrWhiteSpace(extenstion))
+            {
+                switch (extenstion)
+                {
+                    case ".csproj":
+                        supported = StringComparer.OrdinalIgnoreCase.Compare(language, "csharp") == 0;
```

Comment:
Worth creating a language enum?

Reply:
I think eventually we may want this to be open-ended and pluggable at which point the enum would wear out its welcome. Also, the enum doesn't help with authoring the actual Markdown.